spec/models/book_spec.rb

```diff
@@ -21,4 +21,14 @@
       expect(book.reviewed_by?(user_without_review)).to eq false
     end
   end
+  describe "#update_avg_rating" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.